autogluon/utils/tabular/ml/models/tab_transformer/hyperparameters/parameters.py

```diff
@@ -0,0 +1,107 @@
+""" Default (fixed) hyperparameter values used in Neural network model """
```

Comment:
I'm pretty sure these aren't correct. This looks like a copy paste of the mxnet tabular NN model.